### ISSUE TITLE
Custom fields

### ISSUE THREAD

**Issue by eleybourn** [eleybourn/Book-Catalogue#680]
Create 4-10 custom fields (including backup and export) with custom labels from an admin page

**Comment by eleybourn** [comment]
Another request for custom fields - Thanks so much for the quick reply Evan.

I have to apologize, because I meant to include the links from Library Thing in case it helps show what we're hoping for.

On a book's main page at Library Thing,  the left hand menu has the "Work Details" option.  The Lexile level is listed on that page.

Here's an example (main page, then details page):

https://www.librarything.com/work/4044

https://www.librarything.com/work/4044/workdetails

I don't know that Amazon typically lists the Lexile level, but it looks GoodReads' implementation allows readers to "shelve" books with their Lexile designation.  Here's an example of "700" level books at Good Reads:

https://www.goodreads.com/shelf/show/700-lexiles

So at first glance, the Good Reads method seems more generic/free form than Library Thing (I couldn't find the specific Lexile-designation on a book's details page on GR like I could on LT).

In short, I guess our hope would be to have "Lexile level" available in the fields list, and if it's discoverable (via LT or wherever), the field is filled-in, otherwise left blank for us to enter manually.

I hope some of that info helps.

In any case, thank you for your time and consideration of our request (I realize it may not be possible/worthwhile to implement, depending on what you find out).
